### PR TITLE
upgrade ceos to version 4.25.5.1M, fix the issue of arista endless loop

### PR DESCRIPTION
**Pull request by yejianquan**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
1. cEOS containers' mgmt-ip occasionally unavaillable, upgrade cEOS image version will fix that
2. cEOS terminal behavior slightly different from vEOS, enhance arista.py to avoid endless loop 
#### How did you do it?
1. upgrade cEOS image version
2. break loop if ssh connect returns merely an 'exit', as for other commands who relies on the return  e.g. 'show interfaces po1 | json', 'exit' won't appear in input_buffer, so the change won't influence it.
#### How did you verify/test it?
1. Run platform_tests/test_advanced_reboot.py test on physical testbed and it didn't stuck again.
2. Tried run command 'show interfaces po1 | json', it's not influenced.

### CODE REVIEW

**Review by diaryevil — APPROVED** [review]

**Comment by lguohan** [comment]
please investigate the test error

**Comment by yejianquan** [comment]
> please investigate the test error

'show lacp neighbor' command is deprecated by 'show lacp peer', fixed

**Comment by yejianquan** [comment]
Compatible with older EOS version by execute both 'show lacp neighbor' and 'show lacp peer', and select success returned one as result.
Didn't choose to maintain a EOS version-> command mapping, because it's hard to enum all the EOS versions we're using, and will be difficulty to maintian if we will have more EOS versions in the future, the people who upgrade EOS version may not be awared of there's a map to be update. 
If we set 'show lacp peer' as default to avoid the issue, it will hurt backward compatibility.

File: ansible/group_vars/all/ceos.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ceos_image_filename: cEOS64-lab-4.23.2F.tar.xz
-ceos_image_orig: ceosimage:4.23.2F
-ceos_image: ceosimage:4.23.2F-1
```
Comment (lguohan) [inline ansible/group_vars/all/ceos.yml:3]: instead of remove them, maybe we can comment out them, so that users can still uncomment them and use them.

Comment (yejianquan) [reply ansible/group_vars/all/ceos.yml:3]: Greate advice, modified

**Comment by yejianquan** [comment]
It seems that something stuck platform_tests/test_advanced_reboot.py::test_warm_reboot,
I'm looking into it.

**Comment by yejianquan** [comment]
split into three prs to reduce the possible affection, close this one

**Comment by yejianquan** [comment]
split into three prs to reduce the possible affection, close this one

**Pull request closed without merging**